packages/editor/src/plugins/text/components/text-editor.tsx

```diff
@@ -141,7 +143,17 @@ export function TextEditor(props: TextEditorProps) {
 
       <Editable
         readOnly={false}
-        onKeyDown={handleEditableKeyDown}
+        onKeyDown={(e) => {
+          if (e.key === '/') {
+            const text = Node.string(editor)
+            // "Show suggestions when the user types '/' and the editor is empty"
+            if (text.length === 0) {
+              setShowSuggestions(true)
+              e.preventDefault()
+            }
+          }
+          handleEditableKeyDown(e)
+        }}
```

Comment:
Why add inline code, couldn't this all go inside the `useEditableKeydownHandler` hook?

Reply:
Thanks, fixed